kogito-usertasks-custom-lifecycle-quarkus/README.md

```diff
@@ -13,9 +13,32 @@ This example shows
 * working with user tasks
 * four eye principle with user tasks
 	
-	
+* Diagram	
 <p align="center"><img width=75% height=50% src="docs/images/process.png"></p>
+	
+Diagram Properties
+<p align="center"><img src="docs/images/diagramProperties.png"></p>
+
+* Diagram Properties
```

Comment:
should be under `Diagram Properties (bottom)` to be in sync with other docs

Reply:
Done @radtriste